src/app/containers/FrontPageStoryRows/testHelpers.js

```diff
@@ -0,0 +1,17 @@
+import { pathOr, take } from 'ramda';
+import uuid from 'uuid';
+import fixture from '#data/pidgin/frontpage';
+import rtlFixture from '#data/urdu/frontpage';
+
+const getFixtureData = dir => (dir === 'ltr' ? fixture : rtlFixture);
+
+const getPromoFixtures = dir =>
+  pathOr(null, ['content', 'groups'], getFixtureData(dir))
+    .flatMap(group => pathOr(null, ['items'], group))
+    .filter(item => pathOr(null, ['assetTypeCode'], item) === 'PRO')
+    .map(item => ({ id: uuid(), ...item }));
+
+const getNumberPromoFixtures = (dir, number = 1) =>
+  take(number, getPromoFixtures(dir));
```

Comment:
How is `take` vs `slice`?   E.g `getPromoFixtures(dir).slice(0, number);`

Reply:
I think nothing (in this case). Ramda allows for currying, but we aren't using that here.